skyplane/gateway/operators/gateway_operator.py

```diff
@@ -90,7 +90,14 @@ def worker_loop(self, worker_id: int, *args):
                 # TODO: status logging
                 self.chunk_store.log_chunk_state(chunk_req, ChunkState.in_progress, operator_handle=self.handle, worker_id=worker_id)
                 # process chunk
-                succ = self.process(chunk_req, *args)
+                succ = retry_backoff(
+                    partial(
+                        self.process,
+                        chunk_req, 
+                        *args
+                    ),
+                    max_retries=1,
```

Comment:
Running this with `max_retries=1` should make a different from the original code, since it'll just call the function once - are you sure this is what fixed the issue?

Reply:
I‘ve tested it. if the process fails, the sender will put the request chunk back into the cache.

> Running this with `max_retries=1` should make a different from the original code, since it'll just call the function once - are you sure this is what fixed the issue?

Reply:
Yes, but either way I don't think making this changes helps, since either way the function is only called once.